packages/lambda-functions/functions/token/environmentCheck.ts

```diff
@@ -0,0 +1,7 @@
+export function environmentCheck() {
+  return !(
+    process.env.ORGANIZATION_ID === undefined ||
+    process.env.API_KEY === undefined ||
+    process.env.USER_EMAIL === undefined
+  );
```

Comment:
An empty string will be a no-go too I think?
If so:
```suggestion
  return Boolean(
    process.env.ORGANIZATION_ID &&
    process.env.API_KEY &&
    process.env.USER_EMAIL
  );
```

`Boolean` is required for the function to always return a boolean.